x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -1631,18 +1631,21 @@ static void addAuthenticationFieldsToLogEntry(StringMapMessage logEntry, Authent
                     logEntry.with(PRINCIPAL_REALM_FIELD_NAME, creatorRealmName);
                     // No domain information is needed here since API key itself does not work across realms
                 }
-                if (authentication.isRemoteAccess()) {
-                    final Authentication remoteAuthentication = (Authentication) authentication.getAuthenticatingSubject()
+                if (authentication.isCrossClusterAccess()) {
+                    final Authentication innerAuthentication = (Authentication) authentication.getAuthenticatingSubject()
```

Comment:
Just a discussion point and no change is required for the PR: We don't currently have a great name for this authentication object. We should think about its terminology.

Reply:
Indeed. I'll leave this as is for now since I still haven't come up with a better name for it but I agree that we need something better here.